tests/suites/model/metrics.sh

```diff
@@ -8,14 +8,14 @@ run_model_metrics() {
 	file="${TEST_DIR}/test-${testname}.log"
 	ensure "${testname}" "${file}"
 
-	juju deploy ubuntu focal
+	juju deploy ubuntu ubuntu
```

Comment:
is this right? the second arg should be series right?
and below too

Reply:
The 2nd is the application. The app name is optional but we will need the name to do further checking. So I kept the name and rename it from a series name to charm name.